### DISCUSSION TITLE
Machine Learning for the Web : An Introduction to TensorFlow.JS

### DISCUSSION THREAD

**Discussion by shivaylamba** [github/india#83] [Open Source]
Hello Everyone!
Feel free to reach out and ask any questions about TensorFlow.js which is an open source machine learning library for JavasSript developers. 

Whether you are a creative artist or a seasoned machine learning pro, TensorFlow.JS can be used by anyone.

**Comment by carrycooldude** [comment]
Really Excited to see Shivay ❤️

**Reply by shivaylamba** [reply]
Thanks @carrycooldude feel free to ask any questions!

**Comment by akash-pal** [comment]
Hi Shivay,
That was a a very compact and beautifully organized talk.

Definitely will check out the book recommendation. 

What can we expect from the latest versions of tensorflow.js (v3)?
As a beginner is there any difference in regard to this?

**Reply by shivaylamba** [reply]
Thankyou so much @akash-pal 
As a beginner the new versions will be even more user friendly. And the major improvements in the new versions are going to support more pre built models, better performance for pre existing models. And ofcourse the introduction and support for WebGPU as a means of Client Side processing.

**Reply by shivaylamba** [reply]
Website / API: 	tensorflow.org/js
Models: 		tensorflow.org/js/models
Github Code:	github.com/tensorflow/tfjs
Google Group: 	tfjs@tensorflow.org

**Reply by akash-pal** [reply]
Sounds great.
Being a javascript developer this library is a blessing. 
The machine learning terminologies is not easy for a beginner especially when it comes to training your own model.

Being a cricket fan, i always am perplexed by how run outs needs to be checked frame by frame.
If I have to think of this scenario, would this be still possible with tensor flow?
As it might involve things like classification of images which are out or not out.

**Reply by shivaylamba** [reply]
Yes I agree Akash. 
It can be possible by TensorFlow. Yes you might want to create several classes of either the foot coming before the bat or the bat being in line.
